distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -7,3 +9,56 @@ type Function
        the this argument.
     @Builtin_Type
     type Function
+
+
+## An identity function which returns the provided argument.
+
+   Arguments:
+   - x: the value to return.
+
+   > Example
+     five = Function.identity 5 # returns number 5
+
+identity : a -> a
+identity x = x
+
+## Flips the first two arguments of a function. Returns function that
+   takes two arguments, but in opposite order.
+
+   Arguments:
+   - f function that takes two arguments
+
+   > Example
+    Console.log <| Function.flip (+) "world" "hello" # Prints 'helloworld'
```

Comment:
Just a note that currently `Console.log` is not yet supported.

Reply:
OK: [ac4e796](https://github.com/enso-org/enso/pull/3554/commits/ac4e796f99ef055550dad7e83b006f82b53e4df1) - I copied that from the new language examples, but let's assume the update will happen once `Console.log` is introduced.